case/src/main/kotlin/com/ritense/case/web/rest/CaseDefinitionResource.kt

```diff
@@ -182,8 +183,8 @@ class CaseDefinitionResource(
     @RunWithoutAuthorization
     fun import(
         @RequestParam("file") file: MultipartFile
-    ): ResponseEntity<Unit> {
-        return try {
+    ): ResponseEntity<Unit> = runWithoutAuthorization {
```

Comment:
2x runWithoutAuthorization

Reply:
Ah, I overlooked the `@RunWithoutAuthorization`